src/DotNetEnv/IValue.cs

```diff
@@ -46,12 +50,15 @@ public string GetValue ()
     public class ValueCalculator
     {
         public string Value { get; private set; }
+        public string RawValue { get; private set; }
 
-        public ValueCalculator (IEnumerable<IValue> values)
+        public ValueCalculator (IList<IValue> values)
         {
+            RawValue = string.Concat(values.Select(val => val.GetValue(false)));
+            
             // note that we do want this lookup / calculation / GetValue calls in the ctor
             // because it is the state of the world at the moment that this value is calculated
-            Value = string.Join(string.Empty, values.Select(val => val.GetValue()));
+            Value = string.Join(string.Empty, values.Select(val => val.GetValue(true)));
```

Comment:
If you're going to use `string.Concat` above, why not use it here too? It seems like an improvement.